packages/frontend/src/utils/api.ts

```diff
@@ -45,17 +45,18 @@ const handleErrors = (err: AxiosError): void => {
   }
 };
 
+const apiBaseURL = process.env.REACT_APP_API_URL
+  ? `${process.env.REACT_APP_API_URL}/api`
+  : '/api';
```

Comment:
We shouldn't need this conditional -- simply require the env variable be defined via `env-required.ts`

Reply:
See comment above.